lib/ruby_lsp/requests/hover.rb

```diff
@@ -95,6 +96,25 @@ def on_constant_path_node_enter(node)
         generate_hover(node.slice, node.location)
       end
 
+      sig { params(node: Prism::CallNode).void }
+      def on_call_node_enter(node)
+        return if DependencyDetector.instance.typechecker
+        return if node.receiver
```

Comment:
I think we can also consider calls on explicit `self` when the receiver is a `SelfNode`.

Reply:
Good point. Added that.